src/workerd/api/crypto-impl.h

```diff
@@ -267,12 +284,9 @@ const SslDisposer<T, sslFree> SslDisposer<T, sslFree>::INSTANCE;
 #define BIGNUM_new BN_new
 #define BIGNUM_free BN_free
 // BIGNUM obnoxiously doesn't follow the naming convention...
-#define BIO_new BIO_new_mem_buf
-#define BIO_free BIO_free_all
-// Neither does BIO...
 
-#define OSSL_BIO(buf, ...) OSSL_NEW(BIO, buf.begin(), buf.size())
-// Wrap a kj::Array<kj::byte> (or kj::ArrayPtr) with an openssl BIO
+#define OSSL_BIO_MEM() OSSL_NEW(BIO, BIO_s_mem())
+#define BIO_free BIO_free_all
```

Comment:
There is a function named BIO_free in the Openssl API – it might be better to give this a different name and adjust OSSL_BIO_MEM accordingly